packages/codelabs/src/basics/lit-html.md

```diff
@@ -1408,25 +1404,25 @@ render() {
 }
 ```
 
-If you refresh, the UI should remain unchanged. Congratulations, you're already composing elements like a pro :)
+If you refresh, the UI should remain unchanged. If that is the case, congratulations! You're now composing elements like a pro :)
 
-In the template we added, we're seeing something different than before. We're passing along our list of todos to the `todo-list` using an attribute with a `.` in front of it:
+In the template that we added we're seeing something different than before. We're passing along our list of todos to the `todo-list` using an attribute with a `.` in front of it:
 
 ```js
 html`
   <todo-list .todos=${this.todos}></todo-list>
 `;
 ```
 
-This is a special lit-html syntax, where instead of setting an attribute it will set a property on the element. The differences between attributes and properties are something that confuses a lot of people, but it's a very simple concept.
+This is lit-html specific syntax that allows you to set a property on the element instead of an attribute. The differences between attributes and properties often confuses a lot of people, but it's actually a very simple concept to grasp.
```

Comment:
```suggestion
This is lit-html specific syntax that allows you to set a property on the element instead of an attribute. The differences between attributes and properties often confuses a lot of people, but it's a very simple concept.
```